device/isis/segmentrouting.yaml

```diff
@@ -0,0 +1,193 @@
+            MPLS IPv6 Flag. If set, then the router is capable of processing SR-MPLS-encapsulated IPv6 packets on all interfaces.
+          type: boolean
+          default: true
+          x-field-uid: 2
+        srgb_ranges:
+          description: |-
+            This contains the list of SRGB.
+          type: array
+          items:
+            $ref: '#/components/schemas/IsisSR.Srgb'
+          x-field-uid: 3
+        
+    IsisSR.Srgb:
+      description: >-
+        This contains the propeties of Segment Routing Global Block (SRGB) Descriptor.
+        Reference: https://datatracker.ietf.org/doc/html/rfc8667#section-3.1-7.1.1
+      type: object
+      properties:
+        range:
+          description: >-
+            This represents the number of SID in a SRGB range. 
+          type: integer
+          format: uint32
+          default: 8000
+          minimum: 1
+          maximum: 16777215
+          x-field-uid: 1
+        starting_sid: 
+          description: >-
+            The SID/Label sub-TLV contains the first value of the SRGB while the range contains the number of SRGB elements.
+          type: integer
+          format: uint32
+          minimum: 1
+          maximum: 16777215
+          default: 16000
+          x-field-uid: 2
+
+    IsisSR.Srlb:
+      description: |-
+        The SR Local Block (SRLB) sub-TLV contains the range of labels the node has reserved for Local SIDs. 
+        Local SIDs are used, e.g., for Adj-SIDs, and may also be allocated by components other than the IS-IS protocol. 
+        As an example, an application or a controller may instruct the router to allocate a specific Local SID. Therefore, 
+        in order for such applications or controllers to know what Local SIDs are available in the router, 
+        it is required that the router advertises its SRLB.
+        Reference: https://datatracker.ietf.org/doc/html/rfc8667#name-sr-local-block-sub-tlv.
+      type: object
+      properties:
+        range:
+          description: >-
+            This represents the number of SID in a SRGB range. 
+          type: integer
+          format: uint32
+          default: 8000
+          minimum: 1
+          maximum: 16777215
+          x-field-uid: 1
+        starting_sid: 
+          description: >-
+            The SID/Label sub-TLV contains the first value of the SRLB while the range contains the number of SRLB elements.
+          type: integer
+          format: uint32
+          minimum: 1
+          maximum: 16777215
+          default: 16000
+          x-field-uid: 2
+    
+    IsisSR.Algorithm:
+      description: >-
+        Container for SR-Algorithms.
+        Reference: https://datatracker.ietf.org/doc/html/rfc8667#name-sr-algorithm-sub-tlv.
+      type: object
+      properties:
+        algorithm:
+          description: >-
+            The Isis may use various algorithms when calculating
+            reachability to other nodes or to prefixes attached to these
```

Comment:
I think we can should specify the default behaviour for 0 ( regular SPF ) and 1 ( CSPF ) and point to defined value registry if any available.

Reply:
done